packages/devtools_app/lib/src/screens/memory/memory_filter.dart

```diff
@@ -145,32 +145,32 @@ class SnapshotFilterState extends State<SnapshotFilterDialog>
   void didChangeDependencies() {
     super.didChangeDependencies();
 
-    if (controller == widget.controller) return;
-
-    controller = widget.controller;
+    if (_intitialized) return;
```

Comment:
look controller up from `Provider.of<MemoryController>(context)` instead of grabbing it from the widget. See example you just submitted in memory_allocation_table_view

Reply:
it turned out to be not needed after deleting filteredLibrariesByGroupName